client/controllers/graph.controller.js

```diff
@@ -9,111 +9,138 @@
     
     function GraphController(canvasService) {
         var vm = this; // View Model
+        vm.isDrawing = false;
```

Comment:
Changed `canDraw` to `isDrawing` and added another similar variable `isSelecting` for edges.